### ISSUE TITLE
Bug Report:  test_non_air_diluent failing

### ISSUE THREAD

**Issue by erwanp** [radis/radis#588]
### 🐛 Describe the bug

```

@pytest.mark.fast
def test_non_air_diluent(verbose=True, plot=False, *args, **kwargs):

    sf = SpectrumFactory(
        wavelength_min=4200,
        wavelength_max=4500,
        cutoff=1e-23,
        molecule="CO",
        isotope="1,2",
        truncation=5,
        neighbour_lines=10,
        path_length=0.1,
        mole_fraction=0.1,
        medium="vacuum",
        optimization=None,
        verbose=verbose,
    )
    sf.warnings.update(
        {
            "MissingSelfBroadeningWarning": "ignore",
            "NegativeEnergiesWarning": "ignore",
            "LinestrengthCutoffWarning": "ignore",
            "HighTemperatureWarning": "ignore",
            "AccuracyWarning": "ignore",
            "PerformanceWarning": "ignore",
        }
    )

    sf.load_databank("HITRAN-CO", load_columns=["diluent", "equilibrium"])

    # Calculating spectrum for different diluents
    sf.eq_spectrum(Tgas=2000)
    wl1 = sf.df1["hwhm_lorentz"]
    assert sf._diluent == {"air": 0.9}

    sf.eq_spectrum(Tgas=2000, diluent={"CO2": 0.4, "air": 0.5})
    wl2 = sf.df1["hwhm_lorentz"]
    assert sf._diluent == {"CO2": 0.4, "air": 0.5}

    sf.eq_spectrum(Tgas=2000, diluent="CO2")
    wl3 = sf.df1["hwhm_lorentz"]
    assert sf._diluent == {"CO2": 0.9}

    assert (wl1 < wl2).all() and (wl2 < wl3).all()

```

Ends up with 

```
  File "d:\github\radis\radis\lbl\broadening.py", line 335, in pressure_broadening_HWHM
    ** diluent_broadening_coeff["n_" + diluent_molecule.lower()]

KeyError: 'n_co2'


During handling of the above exception, another exception occurred:

Traceback (most recent call last):

(...)

  File "d:\github\radis\radis\lbl\broadening.py", line 353, in pressure_broadening_HWHM
    (airbrd * pressure_atm * diluent["air"])

KeyError: 'air'
```

### 🎲 Radis version

'0.14-g88919c66'

### 💻 Operating system

Windows

**Comment by erwanp** [comment]
Non-air diluent was initially implemented in #495 , and used to work

**Comment by erwanp** [comment]
(tests are OK except from an Exomol molecule mismatch, no need to fix here will be done in #589)
